src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,141 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node<T> first;
+    private Node<T> last;
+
+    private static class Node<T> {
+        T item;
+        Node next;
+        Node prev;
+
+        Node(T item, Node prev, Node next) {
+            this.item = item;
+            this.prev = prev;
+            this.next = next;
+        }
+    }
+
     @Override
     public void add(T value) {
-
+        if (first == null) {
+            first = new Node(value, null, null);
+            last = first;
+            size = 1;
```

Comment:
write size++
after if-else
and then remove them from lines 27 and 32

Reply:
done